openedx/core/lib/api/tests/test_authentication.py

```diff
@@ -69,7 +73,7 @@ class OAuth2AuthenticationDebug(authentication.OAuth2AuthenticationAllowInactive
     url(
         r'^oauth2-with-scope-test/$',
```

Comment:
Should this be `oauth2-with-scope-deprecated-test/` and do we need another `oauth2-with-scope-test/` for the non-deprecated class?

Reply:
As far as I can tell, the scopes part of the code is never used. The tests were copied straight from django-rest-framework-oauth(the repo that we are trying to get rid off). I don't think scopes even need to be tested by us.

Reply:
Do we understand what this comment means? Is this a DOP only issue?
- https://github.com/edx/edx-platform/blob/fbcbb87aa0d518b0bac2d9f58c70cc4d2adefa6d/openedx/core/lib/api/tests/test_authentication.py#L141-L147

And this test?  I think we need to be careful not to introduce potential authn/z bugs in these classes.
- https://github.com/edx/edx-platform/blob/fbcbb87aa0d518b0bac2d9f58c70cc4d2adefa6d/openedx/core/lib/api/tests/test_authentication.py#L141-L147